File: app/models/flight.rb
# == Schema Information
#
# Table name: flights
#
#  id           :integer          not null, primary key
#  flight_num   :integer
#  date         :date
#  origin       :string
#  destination  :string
#  airplanes_id :integer
#  created_at   :datetime         not null
#  updated_at   :datetime         not null
#

class Flight < ActiveRecord::Base
end

File: app/assets/javascripts/models/Flight.js
var app = app || {};

app.Flight = Backbone.Model.extend({
  urlRoot: '/flights',
  defaults: {
    t.integer: "flight_num",
    t.date:    "date",
    t.string:  "origin",
    t.string:  "destination",
    t.integer: "airplanes_id"
  }
});
